### DISCUSSION TITLE
Multiple elements to have the appearance of a card -- how to implement?

### DISCUSSION THREAD

**Discussion by dz4k** [bigskysoftware/missing#17] [General]
I wanted to have `<details>` styling be derived from `missing-card`. This is not possible in any standardized CSS without adding `, details` to card.css, or duplicating styles.

In CSS preprocessors, we have things like extends, but I'd prefer if we stuck to a CSS dialect that will probably be standard in the future.

**Comment by dz4k** [comment]
https://github.com/travco/postcss-extend

This plugin adds Sass-style `@extend`, which seems like what we want.

I couldn't find any action towards a native CSS version of this. There was the `@apply` proposal, but that died out (I didn't like it anyway, but there doesn't seem to be any alternative).

I just want a way to say "pretend that elements which match this selector also match this other selector"

**Comment by geoffrey-eisenbarth** [comment]
Looks like this will eventually be implemented as `@mixin`: https://drafts.csswg.org/css-mixins-1/?hl=en-US#mixin-rule
